### DISCUSSION TITLE
Checking and unchecking "Fully hide Dock" checkmark resets the dock to "left" position

### DISCUSSION THREAD

**Discussion by pofka321** [lawand-dot-io/taskbar#26] [Feature requests]
**Describe the bug**
Checking and unchecking "Fully hide Dock" in "Advanced" preferences check mark resets the dock to "left" position, meaning that settings lost what was set before
**To Reproduce**
1. Use dock in position "Right"
2. Go to taskbar Preferences
3. Click on Advanced
4. Click on "Fully hide Dock"
5. Click on "Fully hide Dock" again
6. See issue

**Expected behavior**
I expected that dock position would not be reset

**Screenshots**
None

**Desktop (please complete the following information):**
 - Taskbar version 1.6
 - OS: 14.2
 - Chip Apple M2

**Additional context**
None

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the bug report. I mentioned in the advanced page that the Dock will be reset to the left, but maybe I should give the user the option to choose, so maybe when you click to un-fully-hide then you see a popup asking if you prefer right or left. The other option is to remember where the dock was before fully-hiding. Maybe you're right and I can do it this way.
